### PR TITLE
Loads database and server configs from TOML config file

### PR DESCRIPTION
**Pull request by briankfitzwater**
- Loads some configs from TOML config file (`config.toml`)
- Database settings can be overridden via `DATABASE_URL` environment variable.  Only tested with sqlite3.  Unlikely to work with other database providers.

### CODE REVIEW

**Comment by briankfitzwater** [comment]
@afeld @Vermyndax What do you think about using [Viper](https://github.com/spf13/viper) instead of writing our own config code?

**Comment by Vermyndax** [comment]
Looks like a fantastic idea to me, especially if we grow this thing out.

File: README.md [hunk 1]
```diff
@@ -35,13 +35,13 @@ This is an API that [creates AWS (sub)accounts in an Organization](https://docs.
 1. Setup the database with Proof-of-Concept data.
 
     ```sh
-    sqlite3 aws-account-broker.db < poc_data.sql  
+    sqlite3 /tmp/aws-account-broker.db < poc_data.sql
```
Comment (afeld) [inline README.md]: I'd keep it local, personally. Fine either way.

File: main.go [hunk 2]
```diff
@@ -7,6 +7,7 @@ import (
 	"os"
 
 	"code.cloudfoundry.org/lager"
+	"github.com/BurntSushi/toml"
```
Comment (afeld) [inline main.go]: Mind running `dep ensure` and checking in the changes?

Comment (briankfitzwater) [reply main.go]: Done.

File: README.md (hunk 1 above)
Comment (briankfitzwater) [reply README.md]: I wanted to ensure the database files didn't get mixed up with the repo.

Comment (afeld) [reply README.md]: I’d just add *.db to the gitignore.

**Pull request merged**